.bash_prompt

```diff
@@ -164,7 +164,14 @@ sexy_bash_prompt_branch_exists () {
 sexy_bash_prompt_parse_git_ahead () {
   # Grab the local and remote branch
   branch="$(sexy_bash_prompt_get_git_branch)"
-  remote_branch="origin/$branch"
+  
+  # remote_branch="origin/$branch"
+  # Compute the remote tracking branch, if no remote branch then echo 0
+  remote_branch="$(git rev-parse --symbolic-full-name "@{upstream}" 2>/dev/null )"
```

Comment:
So I think my last comment didn't get posted.
1. Re Tests: I have no idea how the tests are written or how Jenkins is doing his magic for this project, so one of you will have to help out there.
 
2. Origin as remote when tracking is not defined:
In my opinion, when a user assumes tracking is set up and tries to push (git push or git push origin) the error message spews out this whole message saying:
> The current branch Devel has no upstream branch.
> To push the current branch and set the remote as upstream, use
> git push --set-upstream origin Devel
So IMHO in such a case there is no need for the script to try to guess the remote branch.